js/functions.js

```diff
@@ -0,0 +1,52 @@
+const checkStringLength = function (checkString, i) {
+  if (checkString.length <= i) {
+    return true;
+  } else {
+    return false;
+  }
+};
+checkStringLength('проверяемая строка', 20);
+checkStringLength('проверяемая строка', 18);
+checkStringLength('проверяемая строка', 10);
+
+function checkWord(word) {
+  const splitString = word.split('');
+  const reverseArray = splitString.reverse();
+  const joinArray = reverseArray.join('');
+  if (word.toLowerCase === joinArray.toLowerCase) {
+    return true;
+  } else {
+    return false;
+  }
+}
+checkWord('топот');
+checkWord('ДовОд');
+checkWord('Кекс');
+
+const extractNumber = (string) => {
+  let result = '';
+  for (let i = 0; i < string.length; i++) {
+    if (!Number.isNaN(parseInt(string.at(i), 10))) {
+      result += string.at(i);
+    }
+  }
+  return parseInt(result, 10);
```

Comment:
Лучше форматировать таким образом, return циклы if отделять одной пустой строкой
```suggestion
  let result = '';
  
  for (let i = 0; i < string.length; i++) {
    if (!Number.isNaN(parseInt(string.at(i), 10))) {
      result += string.at(i);
    }
  }
  
  return parseInt(result, 10);
```